index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Tina Wilson - Software Developer</title>
+        <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700|Oswald:400,700" rel="stylesheet">
+        <link rel="stylesheet" href="style.css">
+        <meta name="viewport" content="initial-scale=1.0,width=device-width" />
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link href="https://fonts.googleapis.com/css?family=Merienda|Modern+Antiqua|Roboto+Condensed" rel="stylesheet">
```

Comment:
Consider using only one link to Google Fonts -- combine this with the previous link to there.